### ISSUE TITLE
Primeng 8.1.0 -> ERROR in Couldn't resolve resource ./carousel.css relative to .../carousel.d.ts

### ISSUE THREAD

**Issue by darckyn** [primefaces/primeng#8268]
**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```
**Current behavior**

When I run the command: `ng build --prod`
The following error happens:

**`ERROR in Couldn't resolve resource ./carousel.css relative to /home/.../node_modules/primeng/components/carousel/carousel.d.ts`**

This error only happens when using Primeng  8.1.0  **`ng build --prod`**, using only **`ng build`** the error does not reproduce

Primeng 8.0.3 the error does not reproduce.

**NOTE**: I am not using Carrosel Module in my project.

**Expected behavior**

No error and build for production

**Minimal reproduction of the problem with instructions**

In Primeng version  8.1.0
Just run this command in terminal: ng build --prod

**What is the motivation / use case for changing the behavior?**

Update Primeng 8.0.3 to 8.1.0 

**Please tell us about your environment:**

**OS**: Linux Mint 19.2 Cinnamon
**Kernel**: 4.15.0-66-generic
**IDE**: Visual Studio Code Version 1.39.2

* **Angular version:** 8

* **PrimeNG version:** 8.1.0

* **Browser:** [all]

 
* **Language:** [TypeScript 3.5.3 | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =   v10.17.0

package.json dependences:

```
"dependencies": {
    "@angular/animations": "~8.2.11",
    "@angular/cdk": "^8.2.3",
    "@angular/common": "~8.2.11",
    "@angular/compiler": "~8.2.11",
    "@angular/core": "~8.2.11",
    "@angular/forms": "~8.2.11",
    "@angular/platform-browser": "~8.2.11",
    "@angular/platform-browser-dynamic": "~8.2.11",
    "@angular/pwa": "^0.803.13",
    "@angular/router": "~8.2.11",
    "@angular/service-worker": "~8.2.11",
    "@auth0/angular-jwt": "^3.0.0",
    "@types/googlemaps": "^3.38.0",
    "@types/grecaptcha": "^3.0.0",
    "ajv": "^6.10.2",
    "chart.js": "^2.8.0",
    "core-js": "^3.3.3",
    "flatted": "^2.0.1",
    "font-awesome": "^4.7.0",
    "ngx-countdown": "^8.0.1",
    "node-sass": "^4.12.0",
    "primeicons": "^2.0.0",
    "primeng": "^8.1.0",
    "quill": "^1.3.7",
    "rxjs": "~6.5.3",
    "tslib": "^1.9.0",
    "zone.js": "~0.10.2"
  },
"devDependencies": {
    "@angular-devkit/build-angular": "~0.803.13",
    "@angular/cli": "~8.3.13",
    "@angular/compiler-cli": "~8.2.11",
    "@angular/language-service": "~8.2.11",
    "@types/jasmine": "~3.4.4",
    "@types/jasminewd2": "~2.0.8",
    "@types/node": "^12.11.5",
    "codelyzer": "^5.2.0",
    "jasmine-core": "~3.5.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~4.4.1",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage-istanbul-reporter": "~2.1.0",
    "karma-jasmine": "~2.0.1",
    "karma-jasmine-html-reporter": "^1.4.2",
    "protractor": "~5.4.0",
    "ts-node": "~8.4.1",
    "tslint": "~5.20.0",
    "typescript": "^3.5.3",
    "webpack-bundle-analyzer": "^3.6.0"
  }
```

**Comment by JacobSiegle** [comment]
There was an error in build 8.1.0 according to the blog at https://www.primefaces.org/blog/. Have you tried 8.1.1?

**Comment by darckyn** [comment]
Nice!
The erro was fixed in version 8.1.1.

thx!
